frontend/src/components/InquiryList.js

```diff
@@ -0,0 +1,35 @@
+import styled from 'styled-components/macro'
+
+export default function InquiryList({inquiries}){
+
+    return(
+        <List>
+            {inquiries.map((inquiry) => (
+                <li key={inquiry.id}>
+                    <span>{inquiry.partName}</span>
+                    {/*<Link to={`/user/${user.name}`}>*/}
+                    {/*    <img src={user.avatar} alt={user.name} />*/}
+                    {/*    <span className="user-name">{user.name}</span>*/}
+                    {/*</Link>*/}
+                </li>
+            ))}
+        </List>
+    )
+}
+
+const List = styled.ul`
+  list-style: none;
+  padding: 0;
+  margin: 0;
+
+  a {
+    display: flex;
```

Comment:
not 100% sure but I think from a styling perspective it might make more sense to turn the list item into a display flex  so all items within the list item are displayed the same way